charts/consul/values.yaml

```diff
@@ -117,25 +117,123 @@ global:
   # created by this chart. See https://kubernetes.io/docs/concepts/policy/pod-security-policy/.
   enablePodSecurityPolicies: false
 
-  # Configures Consul's gossip encryption key, set as a Kubernetes secret
+  # secretsBackend is used to configure Vault as the secrets backend for the Consul on Kubernetes installation.
+  # The Vault cluster needs to have the Kubernetes Auth Method, KV2 and PKI secrets engines enabled
+  # and have necessary secrets, policies and roles created prior to installing Consul.
+  # The Vault cluster *must* not have the Consul cluster installed by this Helm chart as its storage backend.
```

Comment:
```suggestion
  # The Vault cluster *must* not have the Consul cluster installed by this Helm chart as its storage backend
  # as that would cause a circular dependency.
```